builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -615,15 +624,15 @@ function store.update_paths()
 	pkgmgr.update_gamelist()
 	for _, game in pairs(pkgmgr.games) do
 		if game.author ~= "" and game.release > 0 then
-			local id = game.author:lower() .. "/" .. game.id
+			local id = game.author:lower():encode_uri_component() .. "/" .. game.id
 			game_hash[store.aliases[id] or id] = game
 		end
 	end
 
 	local txp_hash = {}
 	for _, txp in pairs(pkgmgr.get_texture_packs()) do
 		if txp.author and txp.release > 0 then
-			local id = txp.author:lower() .. "/" .. txp.name
+			local id = txp.author:lower():encode_uri_component() .. "/" .. txp.name
```

Comment:
id is used to track packages locally, and should never be used in a URL as case info is lost